src/Microsoft.TemplateEngine.Cli/HelpAndUsage/HelpForTemplateResolution.cs

```diff
@@ -494,7 +494,7 @@ public static CreationResultStatus HandleParseError(INewCommandInput commandInpu
 
         internal static string GetTemplateHelpCommand(string commandName, ITemplateInfo template)
         {
-            return GetTemplateHelpCommand(commandName, template.ShortName);
+            return GetTemplateHelpCommand(commandName, template.ShortNameList.First());
```

Comment:
If someone didn't list any short names in their template.json, do we catch this error before the execution reaches this line? If not, I suggest we replace this with `FirstOrDefault` so that user won't see an unhandled exception.

Reply:
Having no short name is near to impossible case, I'm not sure if we get here if short name is not set - so much code is relying on short name.
I've added better handling for this case. In future we need better validation for templates to be installed, so engine is not even accepting the templates without mandatory information.